action.yml

```
@@ -51,9 +56,6 @@ runs:
       id: tflint
       shell: bash
       env:
-        # We may want to allow specifying reviewdog and tflint version as
-        # action's input, but let's start with hard coded latest stable version for reviewdog
-        # and latest release for tflint
         REVIEWDOG_VERSION: v0.11.0
```

This would restore the piece of the comment which is still applicable today:
```suggestion
        # We may want to allow specifying reviewdog version as
        # action's input, but let's start with hard coded latest stable version for reviewdog
        REVIEWDOG_VERSION: v0.11.0
```

An unrelated PR specifically for the purpose could suggest removing it

---

You make a good point!

I just saw that the install script we're using for reviewdog supports installing a specific version (and latest is installed otherwise).

I'll add that to this PR and change the title to "_Customizable versions for everything_".

---

I changed my mind on this. 

Reviewdog may have change that break the user flow, so I'd rather keep a hardcoded version for it. I'll apply your comment change to keep it clear.